src/Templating/src/Microsoft.DotNet.Web.ProjectTemplates/RazorComponentsWeb-CSharp.Server.csproj.in

```diff
@@ -12,7 +12,6 @@
     <ProjectReference Include="..\RazorComponentsWeb-CSharp.App\RazorComponentsWeb-CSharp.App.csproj" />
   </ItemGroup>
 
-  <!-- TODO: Remove this block once it's included in the shared framework -->
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Components.Server" Version="${MicrosoftAspNetCoreComponentsPackageVersion}" />
```

Comment:
Spoke about this offline - we cannot remove this package reference since there's a host of packages referenced by Components.Server that are not in the shared fx. This includes the SpaServices packages and SignalR.MessagePack